assets/lokomotive-kubernetes/bootkube/resources/charts/calico/templates/globalnetworkpolicies.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.calico.metadataCIDRs }}
+apiVersion: crd.projectcalico.org/v1
+kind: GlobalNetworkPolicy
+metadata:
+  name: block-metadata-access
+spec:
```

Comment:
Should we include an `order` field? Have you tried to create a policy that allows traffic on top of this?

Reply:
As in the comments, creating `NetworkPolicy` selecting specific pods hole-punches this policy correctly. I think the order only matters between GNPs?